packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountItem/AccountItemContent.tsx

```diff
@@ -146,21 +150,25 @@ export const AccountItemContent = ({
                     {type === 'staking' && <Translation id="TR_NAV_STAKING" />}
                     {type === 'tokens' && <Translation id="TR_NAV_TOKENS" />}
                 </AccountLabelContainer>
-                <BaseCurrency
-                    isLoading={isFiatLoading}
-                    customFiatValue={customFiatValue}
-                    symbol={account.symbol}
-                    formattedBalance={formattedBalance}
-                />
+                {!isCardanoStaking ? (
+                    <BaseCurrency
+                        isLoading={isFiatLoading}
+                        customFiatValue={customFiatValue}
+                        symbol={account.symbol}
+                        formattedBalance={formattedBalance}
+                    />
+                ) : (
+                    <Icon name="check" color={theme.iconPrimaryDefault} size={16} />
```

Comment:
```suggestion
                    <Icon name="check" color={theme.iconPrimaryDefault} size={16} />
```

use variant instead, color is deprecated

Reply:
https://github.com/trezor/trezor-suite/pull/21694/commits/2947158b5eb68e35b2378fef8180ef6a53c8b1ba